packages/admin/resources/views/components/layouts/app/sidebar/item.blade.php

```diff
@@ -48,16 +49,35 @@
             </span>
 
             @if (filled($badge))
-                <span
-                    @class([
-                        'inline-flex items-center justify-center ml-auto rtl:ml-0 rtl:mr-auto min-h-4 px-2 py-0.5 text-xs font-medium tracking-tight rounded-xl whitespace-normal',
-                        'text-primary-700 bg-primary-500/20' => ! $active,
-                        'text-white bg-white/20' => $active,
-                        'dark:text-primary-500' => (! $active) && config('filament.dark_mode'),
-                    ])
-                >
-                    {{ $badge }}
-                </span>
+                <span @class([
+                    'inline-flex items-center justify-center ml-auto rtl:ml-0 rtl:mr-auto min-h-4 px-2 py-0.5 text-xs font-medium tracking-tight rounded-xl whitespace-normal',
+                    'text-white bg-white/20' => $active,
+                    ...match ($badgeColor ?? null) {
```

Comment:
Array unpacking with string keys is [only supported from PHP 8.1](https://wiki.php.net/rfc/array_unpacking_string_keys).